packages/experience-components/package.json

```
@@ -0,0 +1,68 @@
+{
+  "name": "@logto/experience-sample-components",
```

can you align with the directory name?

```suggestion
  "name": "@logto/experience-components",
```